libcodechecker/analyze/analyzer.py

```diff
@@ -178,6 +178,35 @@ def perform_analysis(args, context, actions, metadata):
         statistics_data = manager.dict({'stats_out_dir':
                                         args.stats_output})
 
+    if 'stats_min_sample_count' in args:
+        try:
+            if int(args.stats_min_sample_count) > 1:
+                statistics_data['stats_min_sample_count'] =\
+                    args.stats_min_sample_count
+            else:
+                LOG.error("stats_min_sample_count"
+                          "must be greater than 1.")
+                return
+        except ValueError:
```

Comment:
Using the integer as the argument type it will be checked for valid types by argparse no further check is needed.